client/state/plugins/installed/README.md

```diff
@@ -40,6 +40,14 @@ Get plugins installed on a list of sites (can also be just one site, but it shou
 
 Get the most recent status for a plugin action (including "inProgress" for currently-running actions).
 
+### `getAllPluginsIndexedByPluginSlug( state: Object )`
+
+Get an object for retrieving a plugin by its slug
+
+### `getAllPluginsIndexedBySiteId( state: Object )`
+
+Get an object for retrieving the plugins for a site by a site ID
```

Comment:
This can be a bit confusing since we don't really pass a site ID, rather they're indexed by it, and it's not retrieving the plugins of a site, but rather all plugins indexed by site. Also, is this really about retrieving or not? 

Should we change it to something like:

```suggestion
Get an object of all installed plugins of each site, indexed by site ID.
```